__tests__/news/news.mutation.test.js

```diff
@@ -0,0 +1,260 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+
+let NEWS_ID = '';
+const NEWS_DOES_NOT_EXIST_ID = '1f2ad470eb08183384e6111b';
+describe('test news mutations', () => {
+  test('#1 should add news to database', async () => {
```

Comment:
Also, please, add the test that tries to create an article with duplicated name and gets and error. Please, check that proper error message and status code are returned

Reply:
fixed